src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitSessionController.kt

```diff
@@ -63,12 +63,12 @@ class VisitSessionController(
       example = "MDI",
     )
     prisonCode: String,
-    @RequestParam(value = "prisonerId", required = false)
+    @RequestParam(value = "prisonerId", required = true)
```

Comment:
Made prisonerId mandatory as we do not call this method without passing prisonerId as a parameter.